net-analyzer/bmon/bmon-4.0.ebuild

```
@@ -42,7 +42,5 @@ src_prepare() {
 }
 
 src_configure() {
-	econf \
-		CURSES_LIB="$( $(tc-getPKG_CONFIG) --libs ncurses)" \
-		--docdir="/usr/share/doc/${PF}"
+	econf CURSES_LIB="$( $(tc-getPKG_CONFIG) --libs ncurses)"
```

```suggestion
	econf CURSES_LIB="$("$(tc-getPKG_CONFIG)" --libs ncurses)"
```
The inner command substitution should be quoted as well.

---

added